libs/server/src/lib/utils/read-schematic-collections.ts

```diff
@@ -8,30 +8,47 @@ import {
   listOfUnnestedNpmPackages,
   normalizeSchema,
   readAndCacheJsonFile,
-  readAndParseJson
+  readAndParseJson,
+  toLegacyWorkspaceFormat
 } from './utils';
-import { toLegacyWorkspaceFormat } from './utils';
 
 export async function readAllSchematicCollections(
-  workspaceJsonPath: string,
-  workspaceSchematicsPath: string
```

Comment:
this function didn't seem to be used anywhere else, and it was an easier implementation if I generated the `workspaceSchematicsPath` in this function, rather than sending it in - so that's why I removed the extra param

but I can add it back if anybody thinks it's good to have 👍